### ISSUE TITLE
how to validate in the trainging stage 2 ?

### ISSUE THREAD

**Issue by GraceZhuuu** [XPixelGroup/DiffBIR#154]
`the training script of stage 2 doesn't support validation set`
is here any good suggestion ?
